### ISSUE TITLE
how to build whl with numpy.libs

### ISSUE THREAD

**Issue by wqh17101** [numpy/numpy#19562]
### Reproducing code example:
i want to build whl from source code
so
i run 
```
python setup.py bdist_wheel
```
and i got a whl with 
![image](https://user-images.githubusercontent.com/26429138/126949127-3feac31e-0e2a-4657-9f89-3e8a0f7f9447.png)

but the one from pypi is
![image](https://user-images.githubusercontent.com/26429138/126949254-b3cfed12-d81b-4277-8f5e-6007cbd8fd14.png)

### Error message:
you can see that no numpy.libs in mine
so i wander that how to build whl with numpy.libs

**Comment by mattip** [comment]
You need to tell the build to package BLAS libraries. The numpy wheel build downloads OpenBLAS and sets the OPENBLAS environment variable. One windows it is more complicated [0], but on macOS/linux this is 
```
BUILD_PREFIX=/tmp/openblas
mkdir -p $BUILD_PREFIX
basedir=$($PYTHON_EXE numpy/tools/openblas_support.py)
cp -r $basedir/lib/* $BUILD_PREFIX/lib
cp $basedir/include/* $BUILD_PREFIX/include
export OPENBLAS=$BUILD_PREFIX
```

Or you can use a site.cfg file following the [example](https://github.com/numpy/numpy/blob/main/site.cfg.example)

[0] For windows, you need to create an import library for the renamed DLL

**Comment by Qiyu8** [comment]
AFAIS, The wheel integrated with BLAS is build with [auditwheel](https://github.com/pypa/auditwheel), the command line is `auditwheel repair *.whl`, we recommend using the pre-built manylinux Docker images.

**Comment by mattip** [comment]
right, once you have wheels with a BLAS packaged in, if you them to be used on other machines, you should repackage the wheel with `auditwheel repair` on linux.

**Comment by wqh17101** [comment]
thank you i got the ```numpy.libs```
but more than yours
![image](https://user-images.githubusercontent.com/26429138/127421687-9bac0ef5-01cb-471e-8f39-69e6e6615cc5.png)
what should i do

**Comment by mattip** [comment]
Did you run `autowheel repair` ?

**Comment by wqh17101** [comment]
> Did you run `autowheel repair` ?

auditwheel repair
